flights/backend/controllers/flight_controller.go

```diff
@@ -166,12 +166,21 @@ func BuyTicketForOtherUser(ctx *gin.Context) {
 func SearchFlights(ctx* gin.Context) {
 	httpGin := http.Gin{Context: ctx}
 	var searchFlightsDTO dto.SearchFlightsDTO
-	layout := "2006-01-02T15:04:05.00Z"
+	layout := "2006-01-02"
```

Comment:
I think that we're sending the full DateTime format with yyyy-mm-ddThh:mm:ss.uuuZ, not just the date. Also it would make sense that the departure time is the whole DateTime

Reply:
this is just for the search params, where we search by date and not time